lib/transports/__es__/_base.js

```diff
@@ -74,6 +74,10 @@ class Base {
         if (response.version) {
           this.ESFullversion = response.version.number
           this.ESversion = response.version.number.split('.')[0]
+          if (response.version.distribution && response.version.distribution === 'opensearch') {
+            this.ESversion = 7
+            this.parent.emit('debug', `distribution is opensearch, assuming major version: ${this.ESversion}`)
+          }
           this.parent.emit('debug', `discovered elasticsearch ${prefix} major version: ${this.ESversion}`)
```

Comment:
This block should be wrapped in an else condition, since it should be printed if it's actually elasticsearch